README.md

```diff
@@ -27,15 +27,16 @@ WDFN_SCHEMA_OWNER_USERNAME: "wdfn_owner"
 WDFN_SCHEMA_OWNER_PASSWORD: "changeMe"
 ```
 
-## Testing
-This project contains JUnit tests. Maven can be used to run them (in addition to the capabilities of your IDE).
-
-### Docker Network
-A named Docker Network is needed to run the automated tests via maven. The following is a sample command for creating your own local network. In this example the name is wqp and the ip addresses will be 172.25.0.x
-
-```.sh
+### Running the Demo DB for local development
+The short version:
+```shell
 docker network create --subnet=172.25.0.0/16 wdfn
+docker run -it --network=wdfn -p 127.0.0.1:5437:5432/tcp usgswma/wqp_db:etl
 ```
+The network and the port will need to match the values in the application.yml.
+
+## Testing
+This project contains JUnit tests. Maven can be used to run them (in addition to the capabilities of your IDE).
```

Comment:
Thanks for updating the README. Seems like the headers on lines 38 and 41 are redundant.  Perhaps the
header on line 41 can be kept and line #39 moved down to there.

Reply:
Good point.  I simplified the testing verbiage and added a warning about pointing at a cloud db.